zerver/webhooks/jotform/tests.py

```diff
@@ -15,7 +15,13 @@ def test_response(self) -> None:
 * **Student's Name**: Niloth P
 * **Type of Tutoring**: Online Tutoring
 * **Subject for Tutoring**: Math
-* **Grade**: 12""".strip()
+* **Grade**: 12
+* **Prior Tutoring?** No
+* **Student Id**: ID.png
+* **Report Card**: Report Card Niloth.pdf
+
+
+**Files**: [ID.png](https://www.jotform.com/uploads/UserNiloth/243231271343446/6100762336995170500/ID.png), [Report Card Niloth.pdf](https://www.jotform.com/uploads/UserNiloth/243231271343446/6100762336995170500/Report%20Card%20Niloth.pdf)""".strip()
```

Comment:
Ideally, I think we'd want to avoid having a separate "Files" field, and just link the file values themselves.
But, since a preview of the images are generated, we could list the image names.